components/builder-web/package.json

```diff
@@ -31,14 +31,14 @@
     "travis": "npm run test-unit"
   },
   "dependencies": {
-    "@angular/animations": "^4.3.4",
+    "@angular/animations": "https://registry.npmjs.org/@angular/animations/-/animations-4.3.5.tgz",
```

Comment:
Why this approach and not just a version number pin? I'm sure there's a reason, I'm probably just behind on my reading. 😄

Reply:
You probably want to update the shrinkwrap as well, otherwise this won't take hold. (I'm not seeing the progress indicator locally and I suspect that's why.)